testsuite/features/step_definitions/common_steps.rb

```diff
@@ -700,59 +700,59 @@
 end
 
 When(/^I enable repositories before installing Docker$/) do
```

Comment:
See above, it might be this whole definition has to be destroyed (to be checked).